packages/grafana-ui/src/components/Forms/FieldValidationMessage.tsx

```diff
@@ -11,22 +11,22 @@ export interface FieldValidationMessageProps {
   horizontal?: boolean;
 }
 
-export const getFieldValidationMessageStyles = stylesFactory((theme: GrafanaTheme) => {
+export const getFieldValidationMessageStyles = stylesFactory((theme: GrafanaTheme2) => {
   const baseStyle = `
       font-size: ${theme.typography.size.sm};
-      font-weight: ${theme.typography.weight.semibold};
-      padding: ${theme.spacing.formValidationMessagePadding};
-      color: ${theme.colors.formValidationMessageText};
-      background: ${theme.colors.formValidationMessageBg};
-      border-radius: ${theme.border.radius.sm};
+      font-weight: ${theme.typography.fontWeightMedium};
+      padding: ${theme.spacing(0.5, 1)};
+      color: ${palette.white};
```

Comment:
This looks funky to me. I think this should be `theme.colors.error.text`

Reply:
alright 👍